tasks/prime.py

```diff
@@ -362,6 +369,48 @@ def create_mto_shipment(self, overrides=None):
             self.add_stored(MTO_SHIPMENT, mto_shipment)
             return mto_shipment
 
+    @tag(MTO_SHIPMENT, "createMTOShipment", "expectedFailure")
+    @task
+    def create_mto_shipment_with_duplicate_agents(self, overrides=None):
+        # If moveTaskOrderID was provided, get that specific one. Else get any stored one.
+        object_id = overrides.get("moveTaskOrderID") if overrides else None
+
+        move_task_order = self.get_stored(MOVE_TASK_ORDER, object_id)
+        if not move_task_order:
+            logger.debug("createMTOShipment — expected failure: ⚠️ No move_task_order found")
+            return (
+                None  # we can't do anything else without a default value, and no pre-made MTOs satisfy our requirements
+            )
+
+        agent_type = random.sample(["RELEASING_AGENT", "RECEIVING_AGENT"], 1).pop()
```

Comment:
I think you can use random.choice here instead of sample with pop https://docs.python.org/3/library/random.html#random.choice